tools/symfony.md

```diff
@@ -108,6 +108,13 @@ releases:
     latest: 5.1.3
     lts: false
     link: https://symfony.com/blog/symfony-5-1-0-released
+  - releaseCycle: 5.2
+    release: 2020-11-30
+    support: 2021-07-21
+    eol: 2021-07-21
+    latest: 5.2.3
+    lts: false
+    link: https://symfony.com/blog/symfony-5-2-0-released
```

Comment:
Regarding the `link` entry you can use [https://symfony.com/blog/symfony-5-2-3-released](https://symfony.com/blog/symfony-5-2-3-released). The rest is okay, please update.

Reply:
Done, thanks.